src/zenml/integrations/feast/__init__.py

```diff
@@ -32,7 +32,7 @@ class FeastIntegration(Integration):
     """Definition of Feast integration for ZenML."""
 
     NAME = FEAST
-    REQUIREMENTS = ["feast[redis]>=0.19.4", "redis-server"]
+    REQUIREMENTS = ["feast[redis]~=0.26.0", "redis-server"]
```

Comment:
```suggestion
    REQUIREMENTS = ["feast[redis]~=0.26.0", "redis-server>=6.0.9"]
```

I made the same change in my branch working on fixing the Feast example, but added this. Hope pinning Redis doesn't mess with anything.

Reply:
That shouldn't be a problem. I'll test it, just in case.